avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -293,113 +293,113 @@ Describe 'Pipeline tests' -Tag 'Pipeline' {
 
 Describe 'Module tests' -Tag 'Module' {
 
-    Context 'Readme content tests' -Tag 'Readme' {
+    # Context 'Readme content tests' -Tag 'Readme' {
```

Comment:
Is it intended to have this huge commented block?

Reply:
It is not. I ran the test script about a dozen times locally (for different modules) and to not wait for unrelated tests for length periods of times I commented the test out. Thanks for pointing it out

Reply:
Addressed and moved to #3621